### ISSUE TITLE
Exception when turning to landscape mode

### ISSUE THREAD

**Issue by rspilker** [bennorichters/bnoggles#1]
When during gameplay the screen is rotated from portrait to landscap I get the following output in the console:
```
I/flutter (32302): The following message was thrown during layout:
I/flutter (32302): A RenderFlex overflowed by 426 pixels on the bottom.
I/flutter (32302): 
I/flutter (32302): The overflowing RenderFlex has an orientation of Axis.vertical.
I/flutter (32302): The edge of the RenderFlex that is overflowing has been marked in the rendering with a yellow and
I/flutter (32302): black striped pattern. This is usually caused by the contents being too big for the RenderFlex.
I/flutter (32302): Consider applying a flex factor (e.g. using an Expanded widget) to force the children of the
I/flutter (32302): RenderFlex to fit within the available space instead of being sized to their natural size.
I/flutter (32302): This is considered an error condition because it indicates that there is content that cannot be
I/flutter (32302): seen. If the content is legitimately bigger than the available space, consider clipping it with a
I/flutter (32302): ClipRect widget before putting it in the flex, or using a scrollable container rather than a Flex,
I/flutter (32302): like a ListView.
I/flutter (32302): The specific RenderFlex in question is:
I/flutter (32302):   RenderFlex#50038 relayoutBoundary=up1 OVERFLOWING
I/flutter (32302):   creator: Column ← _InheritedProvider ← Provider ← MediaQuery ← LayoutId-[<_ScaffoldSlot.body>] ←
I/flutter (32302):   CustomMultiChildLayout ← AnimatedBuilder ← DefaultTextStyle ← AnimatedDefaultTextStyle ←
I/flutter (32302):   _InkFeatures-[GlobalKey#91812 ink renderer] ← NotificationListener<LayoutChangedNotification> ←
I/flutter (32302):   PhysicalModel ← ⋯
I/flutter (32302):   parentData: offset=Offset(0.0, 80.0); id=_ScaffoldSlot.body (can use size)
I/flutter (32302):   constraints: BoxConstraints(0.0<=w<=683.4, 0.0<=h<=331.4)
I/flutter (32302):   size: Size(683.4, 331.4)
I/flutter (32302):   direction: vertical
I/flutter (32302):   mainAxisAlignment: start
I/flutter (32302):   mainAxisSize: max
I/flutter (32302):   crossAxisAlignment: center
I/flutter (32302):   verticalDirection: down
```

The screen also displays an error message:
![screenshot_20181022-220335](https://user-images.githubusercontent.com/93318/47316277-06ce3700-d647-11e8-8dea-dbcc1b67236f.png)

**Comment by bennorichters** [comment]
fixed with commit 6e6cd1e11c3d27c06efba57dd8a72ba87629650e .  Only allowing portrait for now. It would be nice to have a more responsive interface and change layout for landscape. ... On the backlog ...

**Comment by bennorichters** [comment]
Only portrait allowed
